native/src/routes/JpalTracking.tsx

```diff
@@ -119,15 +162,19 @@ const JpalTracking = ({ route, theme }: PropsType) => {
 
         <ThemedText theme={theme}>{t('trackingCode')}</ThemedText>
         <Input
+          error={invalid && trackingCode?.length !== TRACKING_CODE_LENGTH}
           value={trackingCode ?? ''}
           onChangeText={updateTrackingCode}
           theme={theme}
           editable={!!trackingEnabled}
           testID='input'
         />
+
+        {error && <ErrorText>{t(error)}</ErrorText>}
```

Comment:
This won't translate the error as the namespace is missing. Should be changed to `{error && <ErrorText>{t(`error:${error}`)}</ErrorText>}`

Reply:
It was included, see above. But not relevant anymore.

Reply:
Well, it was not working, I tested it.

Reply:
Was working for me :thinking: